src/highdicom/sr/templates.py

```diff
@@ -1640,6 +2766,42 @@ def __init__(
             qualitative_evaluations=qualitative_evaluations,
             geometric_purpose=geometric_purpose
         )
+        self[0].ContentTemplateSequence[0].TemplateIdentifier = '1410'
+
+    @property
+    def roi(self) -> Union[ImageRegion, ImageRegion3D, None]:
```

Comment:
Should we have another way to get information about the roi if the roi is encoded "by reference", e.g. a reference to a segment in a segmentation image?

Reply:
What about the following?

* `def image_region(self) -> Union[ImageRegion, ImageRegion3D, None]`
* `def referenced_segment(self) -> Union[ReferencedSegment, ReferencedSegmentationFrame, None]`